openwisp_controller/config/migrations/0015_add_relevant_permissions_to_admin_and_operators.py

```diff
@@ -0,0 +1,66 @@
+from django.db import migrations 
+from django.contrib.auth.models import Permission
+from django.contrib.auth.management import create_permissions 
+from django.db import transaction
+
+def make_default_permissions_in_code(apps, schema_editor):
+    for app_config in apps.get_app_configs():
+        app_config.models_module = True 
+        create_permissions(app_config, apps=apps, verbosity=0)
+        app_config.models_module=None
+    """apps.models_module = True 
+    create_permissions(apps, verbosity=0)
+    apps.models_module = None"""
+
+
+
+
+
+
+
+
+def assignPerm(apps, schema_editor): 
+    Group= apps.get_model('openwisp_users', 'Group')
+    admin = Group.objects.get(name="Administrator")
+    operator = Group.objects.get(name="Operator")
+    operators_and_admins_can_change=["device", "config", "template", "location", "floorplan",]
+    operators_read_only_admins_manage = ["vpn", "ca", "cert",]
+    manage_operations = ["add", "change", "delete"]
+
+    for i in operators_and_admins_can_change:
+        for j in manage_operations:
+            permission=Permission.objects.get(codename="{}_{}".format(j,i))
+
+            admin.permissions.add(permission.pk)
+            operator.permissions.add(permission.pk)
+            
+
+    for i in operators_read_only_admins_manage:
+        try:
+            permission=Permission.objects.get(codename="view_{}".format(i))
+            operator.permissions.add(permission.pk )
+        except Permission.DoesNotExist:
+            pass   
+        
+        for j in manage_operations: 
+            admin.permissions.add(Permission.objects.get(codename="{}_{}".format(j,i)).pk, )
+    
+
+class Migration(migrations.Migration): 
+    dependencies=[
+        ('openwisp_users', '__first__'),
+        ('openwisp_users', '0004_default_groups'),
+        ('pki', '__first__'),
+  #      ('django_loci', '__first__'),
+        ('config', '0014_device_hardware_id'),
+        
+    ]
+
+    operations=[
+        migrations.RunPython(make_default_permissions_in_code),
+        migrations.RunPython(assignPerm),

```

Comment:
add reverse_code=migrations.RunPython.noop as it's done here: https://github.com/openwisp/openwisp-users/blob/master/openwisp_users/migrations/0004_default_groups.py